spec/models/request/sample_compound_aliquot_transfer_spec.rb

```diff
@@ -76,7 +78,20 @@
       expect(sequencing_request.target_asset.aliquots.first.library_type).to eq('Standard')
       expect(sequencing_request.target_asset.aliquots.first.study).to eq(study)
       expect(sequencing_request.target_asset.aliquots.first.project).to eq(project)
+      expect(sequencing_request.target_asset.aliquots.first.library_id).to eq(54)
       expect(sequencing_request.target_asset.samples.first.component_samples.order(:id)).to eq(samples.sort)
     end
+
+    # How the library_id should be set if the source aliquots have different library_ids is not defined
+    # Therefore, set it to null for now, until we have a real requirement
+    context 'with conflicting library_ids' do
+      before { aliquot1.update!(library_id: 82) }
+
+      it 'creates a compound sample with a blank library id' do
+        sequencing_request.transfer_aliquots_into_compound_sample_aliquots
```

Comment:
Could add `expect(sequencing_request.target_asset.aliquots.count).to eq(0)` above here too like the other test, it makes it quite clear what the method is changing.
I guess you could also still check that all the other attributes are being set too: library_type, study, project etc

Reply:
I wanted to keep this one simple so it was testing the specific thing I wanted to test (blank library id), rather than duplicating stuff from the other test and muddying the waters?